apps/trackers/api.py

```diff
@@ -108,20 +105,25 @@ def post(self, request, *args, **kwargs):
                 # component is already suggested for a higher impact flaw/affect -- no-op
                 continue
 
-            offers = ProductDefinitionRules().file_tracker_offers(
+            ps_update_stream = PsUpdateStream.objects.get(name=affect.ps_update_stream)
+            offer = ProductDefinitionRules().file_tracker_offer(
                 affect,
                 impact,
-                ps_module,
+                ps_update_stream,
                 exclude_existing_trackers=exclude_existing_trackers,
             )
-            targets[key] = {
-                "affect": affect,
-                "ps_module": affect.ps_module,
-                "ps_component": affect.ps_component,
-                "streams": list(offers.values()),
-                "impact": impact,
-                "selected": False,  # each module is deselected by default
-            }
+
+            if key in targets and offer is not None:
+                targets[key]["streams"].append(offer)
+            else:
+                targets[key] = {
+                    "affect": affect,
+                    "ps_module": affect.ps_module,
+                    "ps_component": affect.ps_component,
+                    "streams": [offer] if offer is not None else [],
```

Comment:
As an affect is on a stream level, I do not think that this should be left in plural. I know that you probably went the way of less breaking change, when we are changing it anyway I would be for making it as precise as possible.

Reply:
Yeah my goal was to not add a breaking change. But if we are going to add a v2 for this endpoint too, then I will also make some other changes to simplify the API response further.